clevertap-core/src/main/java/com/clevertap/android/sdk/inapp/customtemplates/CustomTemplate.kt

```diff
@@ -18,6 +18,11 @@ class CustomTemplate private constructor(
      */
     val name: String,
 
+    /**
+     * The presenter associated with this template.
+     */
+    val presenter: CustomTemplatePresenter<*>,
```

Comment:
<*> can we avoid the * wildcard here?

Reply:
We can avoid it if we introduce a generic type to CustomTemplate. However that would only move the wildcard usage in the classes that use CustomTemplate regardless of the presenter type.